package.json

```
@@ -8,11 +8,14 @@
     "test": "echo \"No tests defined. Please contribute if you like\" && exit 0",
     "lint": "npx eslint .",
     "start": "cross-env NODE_ENV=development concurrently \"tsc -p tsconfig.electron.json\" \"vite\" \"wait-on -t 30000 http://localhost:54321 && electron ./dist-electron/main.js\"",
-    "build": "cross-env NODE_ENV=production npm run clean && vite build && tsc -p tsconfig.electron.json",
+<
+    "build": "cross-env NODE_ENV=production rimraf dist dist-electron && vite build && tsc -p tsconfig.electron.json",
```

```suggestion
    "build": "cross-env NODE_ENV=production npm run clean && vite build && tsc -p tsconfig.electron.json",
```
`npm run clean` already does what you defined again. Hence change not needed